### DISCUSSION TITLE
Bug while building a Rocky Linux due to wrong when condition

### DISCUSSION THREAD

**Discussion by a-belhadj** [vmware/packer-examples-for-vsphere#949] [Help] [answered]
Bug here: https://github.com/vmware-samples/packer-examples-for-vsphere/blob/e33918344a068eb79c36afaf8119fc6cc2004ccd/ansible/roles/base/tasks/redhat.yml#L36

Ansible doc : You can use logical operators to combine conditions. When you have multiple conditions that all need to be true (that is, a logical and), you can specify them as a list.(https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html)

base_additional_packages are not installed when building a Rocky Linux because it doesnt validate the first condition (distribution == Fedora)

In version 0.20, similar issue occurs, cloud-init installation is always skipped for Rocky

**Comment by tenthirtyam** [comment, marked as answer]
See #950.
